config/locales/vi.yml

```
@@ -21,3 +21,34 @@ vi:
       home_page: "Đây là trang chủ cho"
       application: "ứng dụng mẫu."
       sign_up: "Đăng ký ngay !!!"
+  users:
+    new:
+      sign_up: "Đăng Ký"
+      name: "Tên"
+      email: "Email"
+      password: "Mật Khẩu"
+      confirmation: "Xác nhận mật khẩu"
+      create_my_account: "Tạo tài khoản"
+    index:
+      users: "Người dùng"
+    users:
+      name: "Tên"
+      email: "Email"
+  views:
+    pagination:
+      first: "« ĐẦu"
+      last: "CUỐI »"
+      previous: "‹ TRƯỚC"
+      next: "SAU ›"
+      truncate: "…"
+  controllers:
+    users_controller:
+      welcome: "Chào mừng đến với ứng dụg mẫu!"
+      fail: "Đăng ký thất bại!"
+      user_not_found: "Người dùng không tìm thấy!!!"
+  shared:
+    error_messages:
+      the_form_contains: "Biểu mẫu bao gồm"
+      error:
+        one: "%{count} lỗir"
```

tiếng Việt chi hay hè
```suggestion
        one: "%{count} lỗi"
```